lib/sqlalchemy/orm/attributes.py

```diff
@@ -120,6 +120,7 @@
 
 
 _T = TypeVar("_T")
+_T_co = TypeVar("_T_co", bound=Any, covariant=True)
```

Comment:
**Federico Caselli** (CaselIT) wrote:

not that it changes anything, but `bound=Any` is the default so it can be omitted

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4834

Reply:
Yeah I know but I've seen many places in the code where `_T` was declared like that so didn't want to break any "convention". This file is actually the exception in this regard. Happy to change it back though if you prefer!

Reply:
Let's leave it like this. In needed we can do a cleanup in a separate change